cmd/health-agent/probers.go

```diff
@@ -15,6 +16,7 @@ import (
 
 func setupProbers() (*proberlist.ProberList, error) {
 	pl := proberlist.New("/probers")
+	go func() { pl.Add(aws.New(), "/sys/aws", 0) }()
```

Comment:
Please change the path to /sys/cloud/aws
Unless you think it's better the way it is?